diff --git a/vendor/shieldtech/Android.mk b/vendor/shieldtech/Android.mk
index 2b127dc..16cd7a7 100644
--- a/vendor/shieldtech/Android.mk
+++ b/vendor/shieldtech/Android.mk
@@ -25,18 +25,18 @@ LOCAL_MODULE_SUFFIX        := $(COMMON_ANDROID_PACKAGE_SUFFIX)
 LOCAL_PRIVILEGED_MODULE    := true
 include $(BUILD_PREBUILT)
 
-include $(CLEAR_VARS)
-LOCAL_MODULE               := vendor.nvidia.hardware.shieldtech.inputflinger@2.0-service
-LOCAL_SRC_FILES_64         := $(SHIELD_SHIELDTECH_PATH)/bin64/hw/vendor.nvidia.hardware.shieldtech.inputflinger@2.0-service
-LOCAL_MULTILIB             := 64
-LOCAL_INIT_RC              := etc/init/vendor.nvidia.hardware.shieldtech.inputflinger@2.0-service.rc
-LOCAL_VINTF_FRAGMENTS      := vendor.nvidia.hardware.shieldtech.inputflinger@2.0-service.xml
-LOCAL_MODULE_CLASS         := EXECUTABLES
-LOCAL_MODULE_TAGS          := optional
-LOCAL_MODULE_OWNER         := nvidia
-LOCAL_VENDOR_MODULE        := true
-LOCAL_MODULE_RELATIVE_PATH := hw
-include $(BUILD_NVIDIA_PREBUILT)
+#include $(CLEAR_VARS)
+#LOCAL_MODULE               := vendor.nvidia.hardware.shieldtech.inputflinger@2.0-service
+#LOCAL_SRC_FILES_64         := $(SHIELD_SHIELDTECH_PATH)/bin64/hw/vendor.nvidia.hardware.shieldtech.inputflinger@2.0-service
+#LOCAL_MULTILIB             := 64
+#LOCAL_INIT_RC              := etc/init/vendor.nvidia.hardware.shieldtech.inputflinger@2.0-service.rc
+#LOCAL_VINTF_FRAGMENTS      := vendor.nvidia.hardware.shieldtech.inputflinger@2.0-service.xml
+#LOCAL_MODULE_CLASS         := EXECUTABLES
+#LOCAL_MODULE_TAGS          := optional
+#LOCAL_MODULE_OWNER         := nvidia
+#LOCAL_VENDOR_MODULE        := true
+#LOCAL_MODULE_RELATIVE_PATH := hw
+#include $(BUILD_NVIDIA_PREBUILT)
 
 include $(CLEAR_VARS)
 LOCAL_MODULE               := vendor.nvidia.hardware.shieldtech.rpx@1.0-service
