From 0a8524a624e9c692f392c25d2627bd494302097a Mon Sep 17 00:00:00 2001
From: Thomas <halorocker89@gmail.com>
Date: Sat, 20 Feb 2021 19:26:14 -0500
Subject: [PATCH] tegra-common hack

---
 nvphs/nvphsd.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/nvphs/nvphsd.rc b/nvphs/nvphsd.rc
index 4de7c19..25af4d5 100644
--- a/nvphs/nvphsd.rc
+++ b/nvphs/nvphsd.rc
@@ -18,5 +18,5 @@ service nvphsd /vendor/bin/nvphsd
 # Start nvphsd after boot is complete to allow
 # any filesystems that nvphsd depends on time
 # to initialize.
-on property:dev.bootcomplete="1"
-    start nvphsd
+#on property:dev.bootcomplete="1"
+#    start nvphsd
-- 
2.27.0

