diff --git a/Power.cpp b/Power.cpp
index c93ef48..56ce913 100644
--- a/Power.cpp
+++ b/Power.cpp
@@ -45,7 +45,8 @@ using ::android::hardware::power::V1_0::Feature;
 using ::vendor::nvidia::hardware::power::V1_0::ExtPowerHint;
 
 Power::Power() {
-    pInfo = new powerhal_info();
+    if (!pInfo)
+        pInfo = new powerhal_info();
 
     pInfo->input_devs.push_back({-1, "touch\n"});
     pInfo->input_devs.push_back({-1, "raydium_ts\n"});
diff --git a/nvpowerhal.cpp b/nvpowerhal.cpp
index fe538a4..e1368d8 100644
--- a/nvpowerhal.cpp
+++ b/nvpowerhal.cpp
@@ -577,6 +577,8 @@ void common_power_init(struct powerhal_info *pInfo)
 void common_power_set_interactive(struct powerhal_info *pInfo, int on)
 {
     int dev_id;
+    int i = 0;
+    char name[MAX_CHARS];
     char path[80];
     const char* state = (0 == on)?"0":"1";
 
@@ -605,6 +607,25 @@ void common_power_set_interactive(struct powerhal_info *pInfo, int on)
         }
     }
 
+    while (on)
+    {
+        snprintf(path, sizeof(path), "/sys/bus/serial/devices/serial%d-0/of_node/name", i);
+        if (access(path, F_OK) < 0)
+            break;
+        else {
+            memset(name, 0, MAX_CHARS);
+            sysfs_read(path, name, MAX_CHARS);
+            if (name[strlen(name) - 1] == '\n')
+                name[strlen(name) - 1] = '\0';
+            if (!strncmp(name, "joyconl", MAX_CHARS) || !strncmp(name, "joyconr", MAX_CHARS)) {
+                snprintf(path, sizeof(path), "/sys/bus/serial/devices/serial%d-0/wake", i);
+                ALOGI("Waking up input device: %s", name);
+                sysfs_write(path, state);
+	    }
+        }
+	i++;
+    }
+
     if (pInfo->no_cpufreq_interactive)
         return;
 
